src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -81,7 +81,7 @@ module TextFormat_0_0_4 = struct
       | None ->
           (label_names, label_values)
       | Some (label_name, label_value) ->
-          let label_value_str = Fmt.strf "%a" output_value label_value in
+          let label_value_str = Fmt.str "%a" output_value label_value in
```

Comment:
Since you're changing this line, you might as well use
```suggestion
          let label_value_str = Format.sprintf "%a" output_value label_value in
```